content/release-notes/1.60.0.md

```diff
@@ -0,0 +1,20 @@
+---
+date: 2022-01-25T10:21:28-07:00
+linktitle: "1.60.0"
+title: "1.60.0"
+kubernetes: "1.20, 1.21, and 1.22"
+---
+
+{{<features>}}
+* New user inteface on the Dashboard and Version history pages.
```

Comment:
@ajp-io I did an initial quick pass on this, but let me know when you're done so I can re-review/approve. For this new feature I feel like we're still a bit vague here. I wasn't sure what PM wanted to highlight here, but tossed together the below suggestion from the blog post on it as a jumping off point.

```suggestion
* The admin console app dashboard and version history pages are redesigned so that you can view and sync your license, see the configured snapshot storage destination, take a snapshot, configure automatic updates, and more right from the dashboard.
```

Reply:
Thanks a lot. I pushed a change with some wording we could use. I don't know your opinion on exclamation points in release notes, but I put one in there and linked to the blog with more details.

Reply:
I'm good with that :) Felt like it deserved more love in the RN and we accomplished that. Thanks!